README.md

```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
 
 </div>
 
-## Get running
+---
+
+## About
+
+### What \[is faebryk\]
+
+faebryk is an open-source software-defined EDA (electronic design automation) tool.
+Think of it like the next step from EDA tools like KiCAD, Altium, Eagle...
+in the way those were the next step from designing electronic circuits on paper.
+While the main focus is on the EDA part currently, faebryk aims to become a wholistic system design tool.
+The main concept of faebryk consists of **describing your system** instead of describing its implementation, and iteratively providing implementation details as needed.
+
+### How \[does faebryk work\]
+
+faebryk itself is just a **python library** that you include in your project. It is providing you with all the tools to describe and design your system and to export that design into something useful like for example a netlist, a devicetree, gerbers etc, which you then can use in the next steps of your project. Key concepts of faebryk are the graph, importers, exporters, the library and the user application.
```

Comment:
Tools is confusing.
Emphasize instead that the user writes code with faebryk as a library.
Project is overloaded here, the faebryk/software project and the larger product project.
faebryk is just a python library.
You write code describing your (system/product...) using faebryk (primitives/functionality...)
/You write code describing your ... with the help of faebryk.
Once you have created a design, you can export whatever artifacts you need ...